src/content/okteto-license.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Okteto License
+description: This page provides information about how Okteto's license works for the products we provide.
+sidebar_label: Okteto License
+id: okteto-license
+---
+
+Okteto requires a license key to use any of our paid products. This document explains how our license works for the different products we provide to remove any uncertainty about feature or usage limitations during a trial period or for your chosen tier of paid service.
```

Comment:
`Okteto requires a license key to use any of our paid products` -> as mention latter, this is only needed for Self-hosted.
I would explain what is SaaS and Self-hosted, and then say you need a license for SH or talk to us to try SaaS.

Reply:
Agree. Maybe this fits better in self-hosted section? This is something that admins need to know, but not the regular developer.

Reply:
```suggestion
Okteto requires a license key to use any of our paid products. We offer two paid products: [SaaS and  Self-Hosted](/saas-vs-self-hosted/). If you use Okteto SaaS then no action is required of you to obtain a license as this is fully mnaged by Okteto. If you use Okteto Self-Hosted, then you will need to obtain a license and include that in your `config.yaml` when installing Okteto.

This document explains how our license works for the different products we provide to remove any uncertainty about feature or usage limitations during a trial period or for your chosen tier of paid service.
```

How does this sound @pchico83?

Reply:
Looks great! There is a small type in `mnaged`

Reply:
Fixed and committed 🙇 

Apparently typing at ~125 WPM doesn't matter if every other word is a typo 🙃